static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -150,8 +162,24 @@ export default function SolutionsSection({
             has_streamlined_ui: hasStreamlinedUI,
           }}
         >
-          {t('Open Solutions Hub')}
-          <IconChevron direction="right" size="xs" />
+          {isAutofixSetupLoading ? (
+            <LoadingIndicator mini />
```

Comment:
would a placeholder be better here than a spinner? A spinner might be distracting